netci.groovy

```diff
@@ -2554,6 +2489,30 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
 
                         Utilities.addArchival(newJob, "coreroot.${lowerConfiguration}.zip,testnativebin.${lowerConfiguration}.zip", "")
                     }
+                    else {
+                        assert architecture == 'arm64'
+
+                        // Then, using the same docker image, build the tests and generate the CORE_ROOT layout.
```

Comment:
Nit: this seems incorrect now as we build the tests entirely

Reply:
not sure I understand this comment

Reply:
I misread the commend marking as resolved.